templates/ciena_software_show.tempate

```diff
@@ -0,0 +1,8 @@
+Value VERSION_INSTALLED (\S+)
+Value VERSION_RUNNING (\S+)
+Value KERNEL (\S+)
+
+Start
+  ^\| *Installed Package +\: +${VERSION_INSTALLED}
+  ^\| *Running Package +\: +${VERSION_RUNNING}
+  ^\| *Running Kernel +\: +${KERNEL}
```

Comment:
We require all new templates to ensure capture all known data and raising an error a line is found that is not accounted for. This helps us have some assurance that we are accurately reporting on the data.

```
  ^\+-+
  ^\|\s+
  ^\|\s+Running\s+Package
  ^\|\s+Application\s+Build
  ^\|\s+Package\s+Build\s+Info
  ^\|\s+Running\s+Kernel
  ^\|\s+Running\s+MIB\s+Version
  ^\|\s+(?:Release|Banks)\s+Status
  ^\|\s+(?:Running|Standby) bank
  ^\|\s+Bank\s+package\s+version
  ^\|\s+Bootloader\s+(?:version|status)
  ^\|\s+Bank\s+package\s+version
  ^\|\s+Last\s+(?:command|configuration)\s+file
  ^\s*$$
  ^. -> Error
```

Reply:
Spaces should not be limited unless it is required, as spaces can accidentally increase/decrease between releases. ` ` should be `\s+` or `\s*` depending on what is appropriate.